sycl/test/esimd/vadd.cpp

```diff
@@ -8,6 +8,10 @@
 // RUN: %clangxx -I %sycl_include %s -o %t.out -fsycl \
 // RUN: -fsycl-instrument-device-code
 
+// Adding xfail because cl::sycl::accessor<...>::get_pointer<...>() is not
+// supported in ESIMD context.
+// XFAIL: *
```

Comment:
could you please fix it by converting
```
            simd<int, VL> va = gather<int, VL>(pA + i * VL, offsets);
            simd<int, VL> vb = block_load<int, VL>(pB + i * VL);
            simd<int, VL> vc = va + vb;
            block_store<int, VL>(pC + i * VL, vc);
```
to accessor-based calls or converting
```
      auto accA = bufA.get_access<cl::sycl::access::mode::read>(cgh);
      auto accB = bufB.get_access<cl::sycl::access::mode::read>(cgh);
      auto accC = bufC.get_access<cl::sycl::access::mode::write>(cgh);
```
to USM pointers?

Reply:
Converted to USM pointers.